openapi3/management.yaml

```diff
@@ -14582,6 +14582,57 @@ paths:
             - okta.users.manage
       tags:
         - UserFactor
+  /api/v1/users/{userId}/factors/{factorId}/resend:
+    post:
+      summary: Resend factor enrollment
+      description: Resends a factor challenge (SMS/call/email OTP) as part of an enrollment flow. 
+                   The current rate limit is one OTP challenge (call or SMS) per device every 30 seconds. 
+                   Okta round-robins between SMS providers with every resend request to help ensure delivery of an SMS OTP across different carriers.
+      operationId: resendEnrollFactor
+      parameters:
+        - name: templateId
+          in: query
+          description: id of SMS template (only for SMS factor)
+          schema:
+            type: string
+        - name: userId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: factorId
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Factor
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserFactor'
+        required: true
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserFactor'
+        '400':
+          $ref: '#/components/responses/ErrorApiValidationFailed400'
+        '403':
+          $ref: '#/components/responses/ErrorAccessDenied403'
+        '404':
+          $ref: '#/components/responses/ErrorResourceNotFound404'
+        '429':
+          $ref: '#/components/responses/ErrorTooManyRequests429'
+      security:
+        - apiToken: []
+        - oauth2:
+            - okta.users.manage
+      tags:
+        - UserFactor
```

Comment:
Do the changes shown here propagate down from changes to the spec, or do these changes need to be propagated back up to the spec for everyone else to consume?